bin/replace-secrets.js

```diff
@@ -23,7 +23,7 @@
 const fs = require('fs');
 const path = require('path');
 const YAML = require('yaml');
-const md5 = require('md5');
+const sha1 = require('sha1');
```

Comment:
Hashing functions are available natively in node: https://nodejs.org/api/crypto.html#crypto_class_hash

```js
const crypto = require('crypto');
const hash = crypto.createHash('sha256');

hash.update('some data to hash');
console.log(hash.digest('hex'));
// Prints:
//   6a2da20943931e9834fc12cfe5bb47bbd9ae43489a30726962b576f4e3993e50
```

Reply:
Oh awesome, thanks for letting me know! Done